src/lib/pickles/inductive_rule.mli

```diff
@@ -1,152 +1,133 @@
-module B : sig
-  type t = Impls.Step.Boolean.var
-end
+module type Intf = sig
+  module B : sig
```

Comment:
Does it have to be in the Intf?

Reply:
To avoid changing code outside of pickles, it seems like it needs to be here. But we don't need it in both the Intf and top level module, it is sufficient to just have it in the Intf.